pennylane/vqe/vqe.py

```diff
@@ -174,35 +191,39 @@ def simplify(self):
           (-1) [X0]
         + (1) [Y2]
         """
-        coeffs = []
+        data = []
```

Comment:
I call this `data` because it is a list of scalar tensors, just like `H.data`. Instead, coefficients should be reserved for a  full tensor.

Reply:
Reminder: add tape recorder stop